upup/pkg/fi/cloudup/awstasks/iamoidcprovider.go

```diff
@@ -35,7 +35,7 @@ type IAMOIDCProvider struct {
 	Lifecycle *fi.Lifecycle
 
 	ClientIDs   []*string
-	Thumbprints []fi.Resource
+	Thumbprints []*string
```

Comment:
I thought we needed this for the initial preview... I guess if tests are passing then we don't (or we have a gap in our tests!)

Reply:
We used to because the thumbprint came from the cluster CA cert. Now they are just well-known strings.

I suspect this may be something we can change back in order to support anonymousauth+public apiserver as oidc provider.